### PR TITLE
render advertisementFeatures

### PR DESCRIPTION
**Pull request by webb04**
Render AdvertisementFeatures. Also known as Guardian Labs.

- Use textSans font
- Use labs colours rather than pillar colours
- Remove tags and follow options
- Added lab stories

## Screenshots

Screen Shot 2020-04-28 at 15 03 59

### CODE REVIEW

File: src/components/headline.tsx [hunk 1]
```diff
@@ -76,6 +76,11 @@ const commentStyles = css`
     padding-bottom: ${remSpace[1]};
 `;
 
+const advertisementFeatureStyles = css`
+    ${styles}
+    ${textSans}
```
Comment (JamieB-gu) [inline src/components/headline.tsx:81]: Would it be preferable to use a design system font here?

File: src/components/paragraph.tsx [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
 }
 
-const styles = css`
-    ${body.medium()}
-    overflow-wrap: break-word;
-    margin: 0 0 ${remSpace[3]};
-`;
+const styles = (design: Design): SerializedStyles => {
+    const advertisementFeature = design === Design.AdvertisementFeature
+        ? textSans
```
Comment (JamieB-gu) [inline src/components/paragraph.tsx:19]: Would it be preferable to use a design system font here? E.g. `textSans.medium()`?

```ts
const font = design === Design.AdvertisementFeature ? textSans.medium() : body.medium();

return css`
    ${font}
    overflow-wrap: break-word;
    margin: 0 0 ${remSpace[3]};
`;
```

File: src/components/standfirst.tsx [hunk 3]
```diff
@@ -76,6 +76,11 @@ const media = css`
     }
 `;
 
+const advertisementFeature = css`
+    ${styles}
+    ${textSans}
```
Comment (JamieB-gu) [inline src/components/standfirst.tsx:81]: Would it be preferable to use a design system font here?

File: src/components/advertisementFeature/article.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        margin: 0 auto;
+    }
+`;
+
+
+// ----- Component ----- //
+
+interface Props {
+    imageMappings: ImageMappings;
+    item: Standard | Review;
```
Comment (JamieB-gu) [inline src/components/advertisementFeature/article.tsx:45]: Are these the only two types of articles that can be advertisement features?

File: src/components/headline.tsx (hunk 1 above)
Comment (webb04) [reply src/components/headline.tsx:81]: I had a problem here as I wanted to keep the headline the same size as other articles but that would require larger textSans sizes in the design system https://theguardian.design/2a1e5182b/p/930d69-typography/b/06df96

@SiAdcock

Comment (webb04) [reply src/components/headline.tsx:81]: Would be happy to try and add that to the design system if it's useful? I imagine Dotcom could have the same problem when adding glabs

File: src/components/paragraph.tsx (hunk 2 above)
Comment (webb04) [reply src/components/paragraph.tsx:19]: `textSans.medium()` is a lot smaller than `headline.medium()`. Even the largest textSans is smaller that `headline.medium()`

File: src/components/headline.tsx (hunk 1 above)
Comment (JamieB-gu) [reply src/components/headline.tsx:81]: Is the issue that you need a `34px` version of `Text Sans`, and the largest currently available is `24px`?

Comment (webb04) [reply src/components/headline.tsx:81]: yep, I could go into source and find the rem value used for that but the current way would also do that.

Comment (JamieB-gu) [reply src/components/headline.tsx:81]: I've opened [this](https://github.com/guardian/source/issues/343) on Source.

**Comment by webb04** [comment]
@JamieB-gu The larger `textSans` sizes have been merged into the design system so they should be available from the next release hopefully. 

I've seen us use the older `textSans` in other places to `AdvertisementFeatures` so I've made a ticket to move them all over to using the design system fonts https://theguardian.atlassian.net/browse/AR-25.

Should this be merged beforehand?

**Pull request merged**